pkg/handler/subscriber/wrap_sqs_context.go

```diff
@@ -0,0 +1,49 @@
+package subscriber
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/go-logr/logr"
+)
+
+type SQSCheckerWrappedHandler struct {
```

Comment:
why is this called SQSChecker? what is it checking?

Reply:
It checks if the payload is a SQS event and if so, injects context, I suppose SQSContextInjector might be more appropriate.

Reply:
Its not really injecting the context, its parsing it out right? Something like `SQSWithContextHander`

Reply:
Perhaps I am misunderstanding how "Inject" is used in this context - what I meant was that we are using the context provided by the SQS event. But I like `SQSWithContextHander`

Reply:
did you mean `SQSWithContextHandler`